lib/env-bundler.js

```diff
@@ -298,7 +298,25 @@ const componentResponseProcessor = {
         headers: { Authorization: `Bearer ${token}` },
       });
 
-      contentType.attributes = res.data.payload.attributes;
+      const schema = {
+        'code': 'code',
+        'type': 'type',
+        'names': 'names',
```

Comment:
For sure missing "name" here based on what was extracted. Is that field dead now?